omod/src/main/java/org/openmrs/module/adminui/page/controller/systemadmin/accounts/AccountPageController.java

```diff
@@ -109,9 +117,54 @@ public String post(PageModel model, @MethodParam("getAccount") @BindParams Accou
 	                   @SpringBean("adminService") AdministrationService administrationService,
 	                   @SpringBean("adminUiAccountValidator") AdminUiAccountValidator accountValidator,
 	                   @SpringBean("providerManagementService") ProviderManagementService providerManagementService,
+					   @SpringBean("appFrameworkService") AppFrameworkService appFrameworkService,
 	                   HttpServletRequest request, UiUtils uu) throws IOException {
 		
 		Errors errors = new BeanPropertyBindingResult(account, "account");
+
+        List<Extension> customUserPropertyEditFragments =
```

Comment:
Since some times you are not creating a user, but just updating their details, can we use a convention like? userAccount.userPropertyEditFragment

Reply:
Changed @dkayiwa